nueljoe/healthcare
# feature(forum):implement discussions and comments

#### What does this PR do?
Implements the forum discussions and comments features

#### Description of Task to be completed?
- implement the `GET /discussions` endpoint to fetch a paginated list of discussions

- implement the `POST /discussions` endpoint to create a new discussion

- implememt the `GET /discussions/:id` endpoint to fetch a single discussion

- implement the `DELETE /discussions/:id` endpoint to delete a discussion

- implememt the `GET /discussions/:id/comments` endpoint to fetch a paginated list of comments associated with a given discussion

- implement the `POST /discussions/:id/comments` endpoint to post a comment in a discussion

- implement the `DELETE /discussions/:id/comments/:commentId` endpoint to delete a comment from a discussion

#### How should this be manually tested?
- clone the repository
- cd into the project directory and run `cd api` to CD into the `api` directory
- see the README.md file in the `api` directory for the preliminary steps
- ensure that a mysql database in installed and running on your device
- see the postman documentation on `Forum`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A
#### Questions:
